apps/web/src/pages/integrations/components/ConnectIntegrationForm.cy.tsx

```diff
@@ -116,7 +122,9 @@ it('shows the configuration for the selected provider', () => {
 
   // We may use a for-loop here since order of checks is not important
   for (const cred of credentials) {
-    cy.get(`input[name="${cred.key}"]`).should('have.attr', 'placeholder', cred.displayName);
+    if (cred.key === CredentialsKeyEnum.RequireTls)
+      cy.get(`input[name="${cred.key}"]`).should('have.attr', 'type', 'checkbox');
```

Comment:
This test is failing, might need to review the implementation as something is off.

Reply:
Ah! My bad. Credential type defined in the test case must be `switch` instead of `boolean`.
Fixed it. It should pass now.